doc/source/User_guide/pyaedt_extensions_doc/project.rst

```diff
@@ -0,0 +1,49 @@
+Import Nastran
+==============
+
+Import Nastran or STL file in any 3D modeler application. The user can also preview the imported file and decimate it before import it.
+
+User can access the extension from the icon created in the Automation tab through the extension manager.
+The following image is the extension user interface:
+
+.. image:: import_nastran_ui.png
+  :width: 800
+  :alt: Import nastran UI
+
+User can launch the user interface from the terminal:
```

Comment:
```suggestion
The user interface can also be launched from the terminal:
```

Reply:
You can also launch the extension user interface from the terminal: